android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -264,7 +264,7 @@ protected void onFinishInflate() {
             mComesFromNewTab = BraveActivity.getBraveActivity().isComesFromNewTab();
             mIsFeedLoaded = BraveActivity.getBraveActivity().isLoadedFeed();
             mExistingNewsFeedObject = BraveActivity.getBraveActivity().getNewsItemsFeedCards();
-            mPrevScrollPosition = 1;
+            mPrevScrollPosition = -1;
```

Comment:
maybe it worth to replace `-1`, `-2`  with self-explaining named constants?

Reply:
@AlexeyBarabash good point. replaced